### ISSUE TITLE
Compilation errors seem to be swallowed instead of being reported

### ISSUE THREAD

**Issue by ClemensSchneider** [dherges/ng-packagr#294]
## Type of Issue

```
[x ] Bug Report
[ ] Feature Request
```

## Description

I had setup ng-packagr to run a successful build. Then, after I added some further code to the codebase, suddenly ng-packagr failed building the library stating, that the expected JS file (that should result from the Typescript source) could not be found.

I then tracked down the error to a missing import of an rxjs operator, that was working fine during development (as I move importing of rxjs-operators to a single file).

I found out about the error by dumping the result of the compiler-cli to a file and reading through the compiler-results.

https://github.com/dherges/ng-packagr/blob/1a42ce10dc2bdac02de88a2ea0d5550356e3a57d/src/lib/steps/ngc.ts#L43

So it seems that the compiler-cli does not throw an error but only reports the error as part of the compiler-results. This should be somehow checked/handled to give the library developer a clear error on why the compilation failed.

#### How To Reproduce

Use an rxjs-operator (e.g. map) but do not import it. Then run ng-packagr.

#### Expected Behaviour

An error message should state what the root of the failed build is.

#### Version Information

```
ng-packagr: v2.0.0.rc-0
node: v8.7.0
@angular: v5.0.1
rxjs: 5.5.2
```

**Comment by dherges** [comment]
Hi @ClemensSchneider,

good you reported. It got messed up in the 2.0.0-rc.0 by the switch from `tsc-wrapped` from `compiler-cli`. Please be a little patient as there's a fix for thjis coming in either #279 or #292

**Comment by dherges** [comment]
Fixed by #292 / 2.0.0-rc.2
